src/collector/main.py

```diff
@@ -51,11 +51,11 @@ def update_statistics():
     if is_verified:
         write_json(NETWORK_STATS_FILE_PATH, network_stats)
         logger.info('Network stats are updated')
-        create_db_snapshot()
+        # create_db_snapshot()  # ASH
     else:
         logger.warning('Network stats are invalid')
         reload_db_from_snapshot()
```

Comment:
Instead of always returning True from verify_network_stats_data, better to handle only reload_db_from_snapshot()